src/ui/modules/Hero.tsx

```
@@ -0,0 +1,164 @@
+import { Img } from '@/ui/Img';
+import Pretitle from '@/ui/Pretitle';
+import { PortableText, stegaClean } from 'next-sanity';
+
+// Stats Row Component
+export const StatsRow = ({ stats }: { stats?: Sanity.Stat[] }) => {
+  if (!stats || stats.length === 0) return null;
+
+  return (
+    <div
+      className="mt-10 md:mt-16 flex flex-row flex-wrap justify-start gap-6 md:grid md:grid-cols-4 md:gap-x-8"
+      role="list"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<ol>
<ul>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)